plugins/audio_file_processor/audio_file_processor.cpp

```diff
@@ -253,17 +253,16 @@ void audioFileProcessor::loadSettings( const QDomElement & _this )
 	m_loopModel.loadSettings( _this, "looped" );
 	m_ampModel.loadSettings( _this, "amp" );
 	m_endPointModel.loadSettings( _this, "eframe" );
+	m_startPointModel.loadSettings( _this, "sframe" );
 
 	// compat code for not having a separate loopback point
-	if( _this.hasAttribute( "lframe" ) )
+	if (_this.hasAttribute("lframe") || !(_this.firstChildElement("lframe").isNull()))
```

Comment:
If only loadSettings returned errors this could be something like:
```
if ( m_loopPointModel.loadSettings( _this, "lframe" ) {}
else { m_loopPointModel.loadSettings( _this, "sframe" ); }
```

Reply:
You can simplify it more in that case.
```cpp
if (!m_loopPointModel.loadSettings(_this, "lframe"))
{
	m_loopPointModel.loadSettings(_this, "sframe");
}
```